dashboard/src/components/Togglz.vue

```diff
@@ -0,0 +1,69 @@
+<!--
+  - Copyright (C) 2016-2020 Dennis Neufeld
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as published
+  - by the Free Software Foundation, either version 3 of the License, or
+  - (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<template>
+	<div v-if="isAdmin" class="iframe-container">
+		<div class="loader-container">
+			<div class="is-loading"></div>
+			<iframe class="iframe-child" src="/api/togglz/index"></iframe>
+		</div>
+	</div>
+	<GoBack v-else to="/" />
+</template>
+
+<script>
+import { mapGetters } from "vuex";
+import GoBack from "@/components/GoBack";
+
+export default {
+	name: "Togglz",
+	components: { GoBack },
+	computed: {
+		...mapGetters("user", ["isAdmin"]),
+	},
+};
+</script>
+
+<style scoped lang="scss">
+@import "node_modules/bulmaswatch/darkly/variables";
+/*This achieves a full screen size of the iframe*/
+.iframe-container {
+	position: fixed;
+	height: 85%; //Good enough so that the footer is still visible
```

Comment:
Leaves a big transparent bar at the bottom, which is inconsistent with all other pages.
https://owo.whats-th.is/2c1xRbU.png

Reply:
If the height is 100% it doesn't show the footer, so I eyeballed a percentage that looks okish in the browser and on mobile. If you know a nice way I'm all for adding it but I don't think large time investments can be justified here to make a hacked together administrative page look better^^